cpp/tests/join/point_in_polygon_test_small.cpp

```diff
@@ -172,8 +172,8 @@ TYPED_TEST(PIPRefineTestSmall, TestSmall)
   expect_tables_equivalent(
     cudf::table_view{
       {fixed_width_column_wrapper<uint32_t>(
-         {0, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, 3, 3, 3, 3, 3, 3, 3, 3}),
+         {3, 3, 3, 3, 3, 3, 3, 3, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 3}),
```

Comment:
How was the data for the tests generated? These could be very hard to update / maintain...

Reply:
This was part of the original PR, I assume generated manually.

I changed `cuspatial::join_quadtree_and_bboxes` to sort its output because it's needed by point-to-nearest-polyline, and was thinking predictable sorted output would be more useful for end-users too.